auth-foundation/build.gradle.kts

```diff
@@ -0,0 +1,102 @@
+plugins {
+    id("com.android.library")
+    id("com.google.devtools.ksp")
+    id("kotlin-parcelize")
+    id("org.jetbrains.dokka")
+    id("spotless")
+    kotlin("android")
+    kotlin("plugin.serialization") version libs.versions.kotlin.get()
+    id("com.vanniktech.maven.publish.base")
+}
+
+android {
+    namespace = "com.okta.authfoundation"
+    compileSdk = COMPILE_SDK
+
+    defaultConfig {
+        minSdk = MIN_SDK
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+
+        ksp {
+            arg("room.generateKotlin", "true")
+            arg("room.schemaLocation", "$projectDir/schemas")
+        }
+
+        buildConfigField("String", "SDK_VERSION", "\"okta-auth-foundation-kotlin/$AUTH_FOUNDATION_VERSION\"")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+
+    compileOptions {
+        isCoreLibraryDesugaringEnabled = true
+        sourceCompatibility = SOURCE_COMPATIBILITY
+        targetCompatibility = TARGET_COMPATIBILITY
+    }
+
+    kotlinOptions {
+        jvmTarget = JVM_TARGET
+        freeCompilerArgs += listOf("-Xopt-in=kotlin.RequiresOptIn", "-Xopt-in=com.okta.authfoundation.InternalAuthFoundationApi")
+    }
+
+    buildFeatures {
+        buildConfig = true
+    }
```

Comment:
Could you document this difference in PR description? I am okay with this

Reply:
This was done to remove the InternalVersion.kt. The version is now in the BuildConfig. The kotlinTemplate was just a  SDK_VERSION and it was not showing up in the android project structure in AS.